src/core/Session.ts

```diff
@@ -554,7 +565,9 @@ export default class Session extends EventEmitter {
       visitor_data: options.visitor_data || device_info[13],
       user_agent: options.user_agent,
       client_name: options.client_name,
-      client_version: device_info[16],
+      client_version: Object.values(Constants.CLIENTS).filter(
+        (v) => v.NAME === options.client_name
+      )[0]?.VERSION ?? device_info[16],
```

Comment:
The way this is written means it will always use the version from the constants, instead of the one fetched remotely, but for WEB it should always use the latest version.